packages/orm/src/test/helpers.ts

```diff
@@ -119,18 +126,45 @@ const MOVIES_INITIAL = [
   },
 ];
 
-export class Book extends Model {
+type BookAttrs = {
+  id: string | number;
+  name: string;
+  releaseYear: number;
+  author: any; // Should be TargetModelRelationship<typeof Author, Relations.ForeignKey>
```

Comment:
Using `any` now since `TargetModelRelationship` is not available in this package yet.